src/testing-policies.md

```diff
@@ -1,61 +1,168 @@
 # Testing Policies
 
-Chimera Policies are regular programs built as Wasm modules. Hence policies can
-be testes using the regular testing capabilities offered by the programming
-language of your choice.
+Policies usually need to be tested so we are sure that they behave as
+we expect when deployed on a Kubernetes cluster.
 
-The reference policies we created do that extensively, as seen here:
+Aside from the approach of testing policy logic with the tools that
+your language toolchain already provides, Chimera has a dedicated
+project for testing policies:
+[`chimera-policy-testdrive`](https://github.com/chimera-kube/chimera-policy-testdrive).
 
-  * [pod-runtime](https://github.com/chimera-kube/pod-runtime-class-policy/tree/main/Tests)
-    policy tests
-  * [pod-toleration](https://github.com/chimera-kube/pod-toleration-policy/blob/60388a054550cc94f2116c45c684b3b079bc8090/src/main.rs#L146-L306)
-    policy tests
+The concept of `chimera-policy-testdrive` is quite simple from a user
+point of view. You have to provide:
 
+* The policy to be tested: through the `--policy` argument. At this
+  time you can only load files in the local filesystem. This might
+  change, but it should be good enough for the develop, build, test
+  cycle.
 
-On top of that, Chimera Policies are also "exectutable" programs. Because of
-that it's really easy to run your policies from your shell.
+* A file that contains the admission request object to be provided to
+  the policy. This is a specific Kubernetes operation to be tested
+  against the policy. Provided using the `--request-file` argument.
```

Comment:
I just learnt, the hard way, this is not correct.

The policy-server doesn't give the full `AdmissionReview` object to the policy, it instead gives the dictionary referenced by the `request` key.